android/engine/src/test/java/org/smartregister/fhircore/engine/task/FhirCarePlanGeneratorTest.kt

```diff
@@ -1120,37 +1108,9 @@ class FhirCarePlanGeneratorTest : RobolectricTest() {
             assertTrue(
               tasks.all { it.basedOnFirstRep.reference == carePlan.asReference().reference }
             )
-          }
-          .also { tasks ->
-            val vaccines =
-              mutableMapOf<String, Date>(
-                "BCG" to patient.birthDate,
-                "OPV 0" to patient.birthDate,
-                "PENTA 1" to patient.birthDate.plusDays(42),
-                "OPV 1" to patient.birthDate.plusDays(42),
-                "PCV 1" to patient.birthDate.plusDays(42),
-                "ROTA 1" to patient.birthDate.plusDays(42),
-                "PENTA 2" to patient.birthDate.plusDays(70),
-                "OPV 2" to patient.birthDate.plusDays(70),
-                "PCV 2" to patient.birthDate.plusDays(70),
-                "ROTA 2" to patient.birthDate.plusDays(70),
-                "PENTA 3" to patient.birthDate.plusDays(98),
-                "OPV 3" to patient.birthDate.plusDays(98),
-                "PCV 3" to patient.birthDate.plusDays(98),
-                "IPV" to patient.birthDate.plusDays(98),
-                "MEASLES 1" to patient.birthDate.plusMonths(9),
-                "MEASLES 2" to patient.birthDate.plusMonths(15),
-                "YELLOW FEVER" to patient.birthDate.plusMonths(9),
-                "TYPHOID" to patient.birthDate.plusMonths(9),
-                "HPV 1" to patient.birthDate.plusMonths(108),
-                "HPV 2" to patient.birthDate.plusMonths(114),
-              )
             vaccines.forEach { vaccine ->
-              println(vaccine)
-
-              val task = tasks.find { it.description.startsWith(vaccine.key) }
-              assertNotNull(task)
-              assertEquals(task!!.executionPeriod.start.asYyyyMmDd(), vaccine.value.asYyyyMmDd())
+              val task = tasks.find { it.description.startsWith(vaccine.key) }!!
+              assertEquals(task.executionPeriod.start.asYyyyMmDd(), vaccine.value.asYyyyMmDd())
```

Comment:
this is failing, it was failing before the refactor, 

```
Expected :2021-09-30
Actual   :2021-10-01
```

I can't see where the related logic changed, not sure why the failure

Reply:
Seems like unrelated test. Checking it also

Reply:
oh these are fine on CI, just failing local